internal/webhooks/clusterclass.go

```diff
@@ -197,7 +197,7 @@ func validateVariableUpdates(clusters []clusterv1.Cluster, old, new *clusterv1.C
 	varsDiff := getClusterClassVariablesForValidation(oldVars, newVars)
 
 	errorInfo := errorAggregator{}
-	allClusters := make([]string, len(clusters))
+	allClusters := []string{}
```

Comment:
NOTE: with 1 cluster this resulted in: `allClusters: []string{""}` which was then further appended and eventually lead to an error